generic-helpers/src/main/java/io/elimu/a2d2/helpers/JedisHelper.java

```diff
@@ -60,11 +60,11 @@ protected void connect() {
 	}
 
 	public boolean containsKey(String key) {
-		return jedis.exists(key.getBytes());
+		return jedis.exists(key);
 	}
 
 	public void register(String key, int timeoutInSeconds) {
-		jedis.setex(key.getBytes(), timeoutInSeconds, new byte[] {1,2,3});
+		jedis.setex(key, timeoutInSeconds, "x");
```

Comment:
setex is deprecated
https://redis.io/commands/setex/

Reply:
replaced with call to sex followed by expire